odl/tomo/backends/stir_bindings.py

```diff
@@ -0,0 +1,148 @@
+# Copyright 2014, 2015 The ODL development group
+#
+# This file is part of ODL.
+#
+# ODL is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# ODL is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with ODL.  If not, see <http://www.gnu.org/licenses/>.
+
+
+"""Backend for STIR: Software for Tomographic Reconstruction
+
+Back and forward projectors for PET.
+
+See the STIR `webpage
+<http://stir.sourceforge.net/>`_ for more information.
+"""
+
+# Imports for common Python 2/3 codebase
+from __future__ import print_function, division, absolute_import
+from future import standard_library
+standard_library.install_aliases()
+from builtins import super
+
+from odl.operator.operator import Operator
+
+try:
+    import stir
+    STIR_AVAILABLE = True
+except ImportError:
+    STIR_AVAILABLE = False
+
```

Comment:
Yes, right. Travis didn't complain either. I think there was a problem in CUDA tests before we had the skip decorator, then it would complain about name errors. No longer relevant as it seems.